src/lib/php/Dao/HighlightDao.php

```diff
@@ -196,4 +196,24 @@ public function getHighlightEntries(ItemTreeBounds $itemTreeBounds, $licenseId =
     $highlightEntries = array_merge(array_merge($highlightDiffs,$highlightKeywords),$highlightBulk);
     return $highlightEntries;
   }
+
+  /**
+   * @param licenseMatchId
+   * @return page number
+   */
+  public function getPageNumberOfHighlightEntry($licenseMatchId)
+  {
+    $row = $this->dbManager->getSingleRow(
+      "SELECT (
+                (
+                  SELECT start FROM highlight WHERE fl_fk=$1 ORDER BY start DESC LIMIT 1
```

Comment:
```suggestion
                  SELECT start FROM highlight WHERE fl_fk=$1 ORDER BY start ASC LIMIT 1
```

Can we do a ascending sort so that the user lands at the beginning of the match and manually move forward instead of finding the end of the match and moving backward to find the starting the match? It just feels more natural.